Sources/BraveWallet/Extensions/BraveWalletExtensions.swift

```diff
@@ -259,6 +261,38 @@ extension BraveWallet.NetworkInfo {
     }
     return nil
   }
+  
+  /// Generate the explorer link for the given ERC721 token with the current NetworkInfo
+  func erc721TokenBlockExplorerURL(_ token: BraveWallet.BlockchainToken) -> URL? {
```

Comment:
nit: I think we can combine the two functions? Both pass the same info to the same object 🙂.
`erc721TokenBlockExplorerURL` & `splTokenBlockExplorerURL`